Assets/Scripts/ResourceController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ResourceController : MonoBehaviour
+{
+    // Public vars and serialized fields
+    public Tree treePrefab;
+    public int ClusterRichness
+    {
+        get
+        {
+            return clusterRichness;
+        }
+    }
+    [Range(1, 10)]
+    [Tooltip("A multiplier for the maximum amount of resources per cluster from 1 to 10")]
+    [SerializeField] private int clusterRichness = 5;
```

Comment:
I really like the use of the Range and Tooltip attributes here. However, I think the following format would be a bit more readable and compact:
```suggestion
    [Tooltip("A multiplier for the maximum amount of resources per cluster from 1 to 10")]
    [Range(1, 10)] [SerializeField] private int clusterRichness = 5;
    public int ClusterRichness { get { return clusterRichness; } }
```
I think we could even make putting the property below the field a part of our standard format, just because properties will never have attributes and so will never have any extra lines separating the two, ensuring a more consistent layout for quick skimming of code.